.github/workflows/validation-coding-standards.yml

```diff
@@ -0,0 +1,40 @@
+name: 'Validation / Coding Standards'
+
+on: pull_request
+
+jobs:
+  php:
+    runs-on: ubuntu-latest
+    name: PHP
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Composer Install
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: PHPCS
+        run: composer lint
+
+  css:
+    runs-on: ubuntu-latest
+    name: CSS
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
Be sure to run [Setup Node](https://github.com/actions/setup-node) to ensure the correct node version and dependencies cache is setup correctly: 

```yaml
- name: Setup Node
  uses: actions/setup-node@v3
  with:
    node-version-file: '.nvmrc'
    cache: 'yarn'
```

After-which you many need to install yarn.